docs/src/main/asciidoc/cli-tooling.adoc

```diff
@@ -20,114 +20,151 @@ The Quarkus CLI is currently available as a jar installable using https://jbang.
 ----
-$ qs
+$ quarkus --help
 Usage: quarkus [-ehV] [--verbose] [COMMAND]
-  -e, --errors    Produce execution error messages.
+  -e, --errors    Display error messages.
   -h, --help      Show this help message and exit.
   -V, --version   Print version information and exit.
       --verbose   Verbose mode.
 Commands:
-  build             Build your quarkus project
-  clean             Clean current project
-  create            Create a new quarkus project.
-  create-jbang      Create a new quarkus jbang project.
-  list              List installed (default) or installable extensions.
-  platforms         List imported (default) or all available Quarkus platforms.
-  add               Add extension(s) to current project.
-  remove, rm        Remove an extension from this project.
-  dev               Execute project in live coding dev mode
-  create-extension  Creates the base of a Quarkus extension in different layout depending of the options and environment.
+  create           Create a new project.
+    app            Create a Quarkus application project.
+    cli            Create a Quarkus command-line project.
+  build            Build the current project.
+  dev              Run the current project in dev (live coding) mode.
+  extension, ext   List, add, and remove extensions of an existing project.
+    add            Extension(s) to add to this project.
+    list, ls       List platforms and extensions for this project.
+    remove, rm     Remove extension(s) from this project.
+  completion       bash/zsh completion:  source <(quarkus completion)
+  version          Display version information
 ----
 
-This will display the help information with all the available commands.
-
 [[project-creation]]
 == Creating a new project
 
-To create a new project we use the create-project command:
+To create a new project we use the `create` command
+(the `app` subcommand is implied when not specified):
 
 [source,shell]
 ----
-$ qs create -a myapp
-----
+$ quarkus create
+-----------
 
-This will create a folder called 'myapp' in your current working directory using default
+applying codestarts...
+🔠 java
+🧰 maven
+🗃 quarkus
+📜 config-properties
+🛠 dockerfiles
+🛠 maven-wrapper
+🐒 resteasy-codestart
 
+-----------
+[SUCCESS] ✅ quarkus project has been successfully generated in:
+--> /<output-dir>/code-with-quarkus
+----
 
-groupId, artifactId and version values 
-(groupId='org.acme', artifactId='myapp' and version='1.0.0-SNAPSHOT').
+This will create a folder called 'code-with-quarkus' in your current working directory
+using default groupId, artifactId and version values
+(groupId='org.acme', artifactId='code-with-quarkus' and version='1.0.0-SNAPSHOT').
 
 To specify the groupId, artifactId and version values, 
 use the '--group-id', '--artifact-id' and '--version' options:
 
 [source,shell]
 ----
-$ qs create --group-id com.foo --artifact-id bar --version 1.0 
+$ quarkus create app --group-id com.foo --artifact-id bar --version 1.0
```

Comment:
I thing `  -p, --platform-bom=groupId:artifactId:version` option should be highlighted in the docs too.

Reply:
I've given minimal mention, as this will be expanded/changed once we start dealing w/ streams (which will be preferred, we hope).

Reply:
Added a brief note.. that bit will be revised with inbound streams.. (so I know that comment is probably not as robust as you wanted.. )